docs/fides/docs/installation/configuration.md

```diff
@@ -4,23 +4,53 @@ Fidesctl supports two methods of configuration. The first is via a toml file, an
 
 By default the fidesctl CLI doesn't require a config file and will instead leverage the default values. These are very likely to be wrong however so it is recommended to always configure your settings properly.
 
+
 ## Configuration file
 
-After initializing fidesctl, a default configuration file will be generated and placed within the `.fides` directory. Here's an example of a fidesctl configuration file:
+After initializing fidesctl, a default configuration file will be generated and placed within the `.fides` directory. Here's an example of a default fidesctl configuration file and documentation for each section:
 
-```toml title="fidesctl.toml"
-[api]
-database_url = "postgres:fidesctl@fidesctl-db:5432/fidesctl"
-log_destination = ""
-log_level = 20
-log_serialization = ""
+=== "default fidesctl.toml"
 
-[cli]
-server_url = "http://localhost:8080"
+    ```toml
+    [api]
+    database_url = "postgres:fidesctl@fidesctl-db:5432/fidesctl"
+
+    # Logging
+    log_destination = ""
+    log_level = INFO
+    log_serialization = ""
+
+    [cli]
+    local_mode = False
+    server_url = "http://localhost:8080"
+
+    [user]
+    encryption_key = "test_encryption_key"
+    ```
```

Comment:
This doesn't feel logically connected to the config option tables, so it should probably just live inline. This will also help readers compare the values in the tables to the values here.